contracts/StablecoinConverter.sol

```diff
@@ -91,12 +98,13 @@ contract StablecoinConverter is EpochTokenLocker {
     /** @dev Constructor determines exchange parameters
       * @param maxTokens The maximum number of tokens that can be listed.
       * @param _feeDenominator fee as a proportion is (1 / feeDenominator)
-      * @param feeToken Address of ERC20 fee token.
+      * @param _feeToken Address of ERC20 fee token.
       */
-    constructor(uint maxTokens, uint128 _feeDenominator, address feeToken) public {
+    constructor(uint maxTokens, uint128 _feeDenominator, address _feeToken) public {
```

Comment:
nit _feeToken should probably be of type TokenOWL and then downcasted below to address, rather than the other way around.